common/controllers/form-wizard.js

```diff
@@ -17,14 +18,37 @@ class FormController extends Controller {
     next()
   }
 
+  setupArbitraryMultipleFields(req, res, next) {
```

Comment:
I wonder if we can find a better name for this method...

> **arbitrary**
> based on random choice or personal whim, rather than any reason or system.

☝️ that use of this doesn't really fill me with confidence in this method...

Reply:
Arbitrary in that it depends on the personal whim of the user.

`setupExplicitNumberOfMultipleFields`?

Reply:
as it no longer applies just to `POST`s and simply sets the values... it's now `setupAddMultipleFieldsValues`